L1Trigger/DemonstratorTools/interface/GTTInterface.h

```diff
@@ -0,0 +1,100 @@
+#ifndef L1Trigger_DemonstratorTools_GTTInterface_h
+#define L1Trigger_DemonstratorTools_GTTInterface_h
+
+#include <cstddef>
+
+#include "L1Trigger/DemonstratorTools/interface/LinkId.h"
+#include "L1Trigger/DemonstratorTools/interface/ChannelSpec.h"
+#include "L1Trigger/DemonstratorTools/interface/EventData.h"
+#include "L1Trigger/DemonstratorTools/interface/FileFormat.h"
+#include "L1Trigger/DemonstratorTools/interface/Frame.h"
+
+namespace l1t::demo {
+  // map of logical channel ID -> [TMUX period, interpacket-gap & offset; channel indices]
+  typedef std::map<LinkId, std::pair<ChannelSpec, std::vector<size_t>>> ChannelMap_t;
+
+  static constexpr size_t kFramesPerTMUXPeriod = 9;
```

Comment:
I think some of these static consts should either be scoped within a `gtt` namespace, or have GTT added to their names, to avoid any chance of ambiguity.

Reply:
At the moment they are down in l1t::demo namespace, that could be l1t::demo::gtt instead. GTT in the name could be clearer across subsystems than different encapsulating scope, however. @aloeliger @epalencia any strong feelings on this (I don't)?

Reply:
I don't either frankly as long as they are under some `l1t` namespace and not global.